src/lldp_syncd/daemon.py

```diff
@@ -270,9 +270,9 @@ def parse_update(self, lldp_json):
                                         'lldp_loc_sys_name',
                                         'lldp_loc_sys_desc',
                                         'lldp_loc_man_addr')
-                    parsed_chassis = dict(zip(loc_chassis_keys,
+                    parsed_chassis = dict(list(zip(loc_chassis_keys,
```

Comment:
This `list()` conversion may also be unnecessary. Need to confirm.

Reply:
@jleveque Updated